src/actinia_core/core/common/process_chain.py

```diff
@@ -189,6 +189,47 @@ def _process_chain_to_process_list(self, process_chain):
 
         return downimp_list
 
+    def _get_stac_import_download_commands(self, entry):
+        """Helper method to get the stac import and download commands.
+
+        Args:
+            entry (dict): Entry of the import description list
+
+        Returns:
+            stac_commands: The stac download and import commands
+        """
+        # Check for band information
+
+        stac_source = entry["import_descr"]["source"]
+
+        if "semantic_label" in entry["import_descr"]:
+            stac_semantic_label = entry["import_descr"]["semantic_label"]
+
+        if "extent" in entry["import_descr"]:
+            if "spatial" and "temporal" not in entry["import_descr"]["extent"]:
+                raise AsyncProcessError("Unknown spatial or/and temporal parameters"
+                                        "in the process chain definition")
+
+            if "spatial" in entry["import_descr"]["extent"]:
+                if "bbox" in entry["import_descr"]["extent"]["spatial"]:
+                    stac_extent = entry["import_descr"]["extent"]
+
+            if "temporal" in entry["import_descr"]["extent"]:
+                if "interval" in entry["import_descr"]["extent"]["temporal"]:
+                    stac_extent = entry["import_descr"]["extent"]
```

Comment:
In this regard, both interval and bbox are required (mandatory) to filter the STAC collections, right? I separated them as it might be more flexible to the user to filter space or time.

Reply:
Ok then leave as is, fine for me too.